src/requesthandler/RequestHandler_General.cpp

```diff
@@ -248,7 +249,8 @@ RequestResult RequestHandler::TriggerHotkeyByName(const Request &request)
 	if (!request.ValidateString("hotkeyName", statusCode, comment))
 		return RequestResult::Error(statusCode, comment);
 
-	obs_hotkey_t *hotkey = Utils::Obs::SearchHelper::GetHotkeyByName(request.RequestData["hotkeyName"]);
+	obs_hotkey_t *hotkey = Utils::Obs::SearchHelper::GetHotkeyByName(
+		request.RequestData["hotkeyName"], request.Contains("contextName") ? request.RequestData["contextName"] : "");
```

Comment:
To maintain code consistency, please implement the optional context name field like done here: https://github.com/obsproject/obs-websocket/blob/ee2e24e42f36faedbca486c7a308444c0f3c3fc7/src/requesthandler/RequestHandler_Inputs.cpp#L40

(I understand it seems a bit wordy, but I would rather change every similar instance to something else in the future than have a bunch of different implementations of the same logic sequence)